### DISCUSSION TITLE
Move config and getters out of content folder?

### DISCUSSION THREAD

**Discussion by sbusso** [flowershow/flowershow#423] [General and Help]
Currently, `config.js` and getters are expected to be in the content folder. This is not ideal as it is mixed with the notes and would be better at root level for config and lib or scripts for getters.

**Comment by olayway** [comment]
Hi, @sbusso! Thank you for your feedback! I've just added this to a list of self-publishing UX improvement ideas in this issue: #352.

**Reply by sbusso** [reply]
thanks @olayway. If those changes are integrated to flowershow code, what is the process to update existing setups?

**Reply by olayway** [reply]
@sbusso the changes have not been made yet. I've just added it to our list of improvements (or ideas for improvements) that we'll work on in the future.

**Comment by rufuspollock** [comment]
I agree with this suggestion. I've regretted that choice of mine from early on.
